ipamutils/utils.go

```diff
@@ -48,3 +60,75 @@ func initGranularPredefinedNetworks() []*net.IPNet {
 	}
 	return pl
 }
+
+// InitAddressPools allows to initialize the local and global scope predefined
+// address pools to the desired values. It fails is invalid input is passed
+// or if the predefined pools were already initialized.
+func InitAddressPools(list []*PredefinedPools) error {
```

Comment:
This method I guess is a very good candidate for a simple go test that covers the proper allocation of the pools, please add it